galaxychop/core.py

```diff
@@ -507,27 +457,26 @@ def jcirc(self, bin0=0.05, bin1=0.005):
         Parameters
         ----------
         bin0 : float, default=0.05
-            Size of the energy bin of the inner part of the galaxy, in the
-            range of (-1, -0.1) of the normalized energy.
-
+            Size of the specific energy bin of the inner part of the galaxy,
```

Comment:
Some parameters description start with capitals and others not. You must normalize that